fathom/src/surface/grammar.lalrpop

```diff
@@ -199,7 +199,7 @@ ProjTerm: Term<'arena, ByteRange> = {
 };
 
 AtomicTerm: Term<'arena, ByteRange> = {
-    "(" <term: Term> ")" => term,
+    <start: @L> "(" <terms: Seq<Term, ",">> ")" <end: @R> => if terms.len() == 1 { terms[0].clone() } else { Term::Tuple(ByteRange::new(file_id, start, end), terms) },
```

Comment:
This means that `"(" <Term> "," ")"` gets parsed as a parenthesized term. Unsure if this is what we want. I think maybe we should have separate rules for: `"(" ")"`, `"(" <Term> ")"`, `"(" <Term> "," ")"` and `"(" <Term> "," <Seq<Term, ",">> ")"`?

Reply:
Sure!